src/main/java/net/coderbot/iris/gui/element/ShaderPackListWidget.java

```diff
@@ -0,0 +1,125 @@
+package net.coderbot.iris.gui.element;
+
+import com.mojang.blaze3d.systems.RenderSystem;
+import net.coderbot.iris.Iris;
+import net.coderbot.iris.gui.GuiUtil;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.font.TextRenderer;
+import net.minecraft.client.gui.widget.AlwaysSelectedEntryListWidget;
+import net.minecraft.client.util.math.MatrixStack;
+import net.minecraft.text.LiteralText;
+import net.minecraft.text.MutableText;
+import net.minecraft.util.Formatting;
+import net.minecraft.util.Tickable;
```

Comment:
`TickableElement` is a thing of you are checking for an element which may tick.

Reply:
Fixed, will commit soon

Reply:
Apparently extending `TickableElement` does not guarantee that child elements will be ticked...